tablepyxl/style.py

```diff
@@ -136,17 +146,15 @@ class Element(object):
     """
     def __init__(self, element, parent=None):
         self.element = element
+        self.number_format = None
         parent_style = parent.style_dict if parent else None
         self.style_dict = StyleDict(style_string_to_dict(element.get('style', '')), parent=parent_style)
-        self._style_cache = None
 
     def style(self):
         """
         Turn the css styles for this element into an openpyxl NamedStyle.
         """
-        if not self._style_cache:
-            self._style_cache = style_dict_to_named_style(self.style_dict)
-        return self._style_cache
+        return style_dict_to_named_style(self.style_dict, number_format=self.number_format)
```

Comment:
I was going to say yes, but I tested each 10x with a large example file just now, and keeping this in place is consistently 4-5% faster.  Didn't seem noticeable initially, but it does help.  Back it goes.